pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_content_checklist.adoc

```diff
@@ -0,0 +1,365 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+
+[id='ref_content_checklist_{context}']
+== Content Checklist for {ProductName}
+
+[role="_abstract"]
+In order to be sure that every component of the {ProductName} tool chain can work with your content, be sure it complies with the following guidelines:
+
+.Assembly requirements
+[cols="30%,35%,35",options="header"]
+|====
+|Requirement
+|Description
+|Example
+
+|Avoid unsupported includes
+
+|The only supported structure for an include is: `include::path/to/some/module/some-module.adoc[leveloffset=+x]`
+
+Always have an empty line between includes.  This applies to both master.adoc and other assemblies.
+|*Supported*
+`include::meta/enterprise/meta/proc_providing-feedback-on-red-hat-documentation.adoc[]`
+
+`include::modules/con_concept-module.adoc[leveloffset=+1]`
+
+`include::assemblies/some_assembly.adoc[leveloffset=+2]`
+
+*Not supported*
+
+`:leveloffset: +1
+modules/core-services/proc_registering-the-system-after-the-installation.adoc[leveloffset: -1]`
```

Comment:
Looks like `include::` is missing at the beginning of this line

Reply:
@noopurmath3 , it does look like 'include::' is missing on line 34

Reply:
I copied what was written in https://docs.engineering.redhat.com/pages/viewpage.action?pageId=173289387 -- see the "Avoid Unsupported includes" section. I can check the person who wrote the document or if there is text you;d suggest I can add it.